### ISSUE TITLE
 using bert pretrainning there are so many trainable params?

### ISSUE THREAD

**Issue by liuchenbaidu** [CyberZHG/keras-bert#108]
i imitate keras-bert/demo/tune/keras_bert_classification_tpu.ipynb for test classfication in GPU.then
i get the params:
Total params: 101,973,890
Trainable params: 101,973,890
after one epoch,the acc is 0.89

if i frozening the bert_model params, the params is :
Total params: 101,973,890
Trainable params: 283,890
after one epoch,the acc is 0.15 

why does it happend?

**Comment by CyberZHG** [comment]
Because the original `NSP` is not suitable for the task.
(However, it's wired to get an accuracy score less than 0.5)
